packages/runtime/src/jest-setup.ts

```diff
@@ -1,12 +1,16 @@
 import '@testing-library/jest-dom'
-import { matchers } from '@emotion/jest'
+import { matchers, createSerializer } from '@emotion/jest'
 import { server } from './mocks/server'
 
 beforeAll(() => server.listen())
 afterEach(() => server.resetHandlers())
 afterAll(() => server.close())
 
 expect.extend(matchers)
+if (typeof window !== 'undefined') {
+  // jest-environment jsdom
+  expect.addSnapshotSerializer(createSerializer())
```

Comment:
Emotion serializer requires `jsdom` Jest env, add it only if running in that env.